### PR TITLE
Document category mounts

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/ApiOverview/Categories/Index.rst [hunk 1]
```diff
@@ -132,3 +132,16 @@ for rendering. Check out the
 
 The HMENU object also has a :ref:`"categories" special type <t3tsref:hmenu-special-categories>`,
 to display a menu based on categorized pages.
+
+User permissions for system categories
+======================================
+
+In most aspects system categories are treated like any other record. They can
+be viewed or edited by editors if they are stored in a folder where the editor
+has access to and if the table :sql:`sys_category` if allowed in the field
```
Comment (liayn) [inline Documentation/ApiOverview/Categories/Index.rst:141]: .. and if the table :sql:`sys_category` is allowed in the field ... of an assigned user group

Comment (mbrodala) [reply Documentation/ApiOverview/Categories/Index.rst:141]: You can make suggestions ready for commit. ;-)

Comment (mbrodala) [inline Documentation/ApiOverview/Categories/Index.rst:141]:
```suggestion
has access to and if the table :sql:`sys_category` is allowed in the field
```

**Review by mbrodala — APPROVED** [review]

File: Documentation/ApiOverview/Categories/Index.rst (hunk 1 above)
Comment (liayn) [reply Documentation/ApiOverview/Categories/Index.rst]: Uh.. never realized that. Thanks for the info. (I'm rarely doing reviews on github, it still does not feel "right" 😀 )

**Pull request merged**